src/IssueViz.Security.UnitTests/Commands/HotspotsToolWindowCommandTests.cs

```diff
@@ -19,32 +19,33 @@
  */
 
 using System;
-using System.Collections.Generic;
 using System.ComponentModel.Design;
 using FluentAssertions;
 using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Moq;
+using Moq.Protected;
 using SonarLint.VisualStudio.Integration;
-using SonarLint.VisualStudio.Integration.UnitTests;
 using SonarLint.VisualStudio.IssueVisualization.Security.Commands;
 using SonarLint.VisualStudio.IssueVisualization.Security.HotspotsControl;
+using ThreadHelper = SonarLint.VisualStudio.IssueVisualization.Security.UnitTests.Helpers.ThreadHelper;
 
 namespace SonarLint.VisualStudio.IssueVisualization.Security.UnitTests.Commands
 {
     [TestClass]
-    public class HotspotsToolWindowCommandTests : ToolWindowCommandTests<HotspotsToolWindow>
+    public class HotspotsToolWindowCommandTests
```

Comment:
I ended up deleting the common suite.
In order to use it, I would've had to add the shell-specific tricks to TestsInfrastructure, which spread to the other test projects. It wasn't worth it IMO.